dataStandard_versioned_docs/version-5/whats-new/whats-new-v52.md

```diff
@@ -0,0 +1,204 @@
+
+_Assessment Administration_ is to refer a wide range of activities involved in procuring, organizing, delivering, and scoring assessments for a student population. It includes
+
+* identifying the target student group,
+* setting the assessment window dates,
+* selecting participating schools for delivery and proctoring,
+* ensuring the availability, functionality and security of technologies used for assessment delivery,
+* managing student registration logistics and communication with the assessment vendor,
+* accommodating special needs of students,
+* handling the logistics of scoring and returning assessment results to educational organizations.
+
+Education organizations involved in the assessment administration may have different roles as assigning, participating, testing, and reporting.
+
+An _Assessment Battery Part_ is a module or collection of assessment tests or questions designed for student delivery that can be administered at different times, scored separately or differently, taken by various groups of students, or require distinct logistics or accommodations.
+
+_Assessment Accommodations_ entail altering the design or delivery of assessments to cater to the needs of students with disabilities or English learners (EL) who cannot effectively take standard assessments. These accommodations can differ depending on the assessment part, such as using a translator for ELA or a calculator for Mathematics. Some special education students may need additional modifications to participate in assessments.
+
+Use cases Ed-Fi has discussed with the community members revealed that a typical process of assessment registration in the field has following steps
+
+* Education organizations collaborate with the assessment vendor to determine the time frame and guidelines for administering an assessment.
+* The specifics of assessment participation and delivery are outlined.
+* Students are registered for the administration of the assessment.
+* The assessment vendor receives the registration list of students along with selected student information.
+
+#### Model Details
+
+To allow community members track data in the Ed-Fi UDM, Assessment Registration domain is created with the following entities and associations:
+
+<u>_AssessmentAdministration_:</u> It is to store data for the administration of an assessment. It has references to AssigningEducationOrganization and Assessment entities, and tracks information relating to AdministrationIdentifier, AssessmentAdministrationPeriod and AssessmentBatteryPart.
```

Comment:
We usually don't utilize underlining in this document, only italics or bold. We don't have to change it right now, but I recommend changing in the future. To be consistent with other areas of the documentation, I would use bold only, without italics or underline.

Reply:
I have started as bold only but the font size in bold was the same with the font size of the header this is part of that is why I have switched to the the underlined and italicized.